src/utils/statuses/getStatusBadgeCopy.ts

```diff
@@ -1,7 +1,7 @@
 // &#8209; is a non-breaking hyphen used so that statuses with a hyphen do not word break when the page width shrinks
 export const getStatusBadgeCopy = (status: string) =>
```

Comment:
I think this function becomes a little confusing to read

It might be simpler if we just did an undefined check like `if (status) { // do this stuff } else { return null; }`